src/message/messageActionSheet.js

```diff
@@ -16,18 +16,24 @@ const copyToClipboard = async ({ message, auth }) => {
   Clipboard.setString(rawMessage);
 };
 
-export const actionSheetButtons = [
-  'Narrow to conversation',
-  'Reply',
-  'Copy to clipboard',
-  'Cancel'
+const editMessage = async ({ message, auth, startEditMessage }) => {
+  startEditMessage(auth, message.id);
+};
+
+const isSentByYouAndNarrowed = (message, auth, narrow) =>
+  isSentByYou(message, auth) && !isHomeNarrow(narrow);
+
+const actionSheetButtons = [
```

Comment:
Nice, I was thinking to add exactly that, but decided not to for now. Still like it a lot ;)